src/osuTK.Android/AndroidGameView.cs

```diff
@@ -537,14 +534,11 @@ private void RunIteration (CancellationToken token)
             if (!ReadyToRender)
                 return;
 
-            curUpdateTime = DateTime.Now;
-            if (prevUpdateTime.Ticks != 0) {
-                var t = (curUpdateTime - prevUpdateTime).TotalSeconds;
-                updateEventArgs.Time = t;
-            }
+            if (watchUpdate.ElapsedTicks != 0)
+                updateEventArgs.Time = (double)watchUpdate.ElapsedMilliseconds / 1000;
 
             UpdateFrameInternal (updateEventArgs);
-            prevUpdateTime = curUpdateTime;
+            watchUpdate.Restart();
```

Comment:
This is wrong and will potentially lose time each loop (where did the time go between the call to `ElapsedMilliseconds` and this?). Store the last value and compare the difference instead.

Reply:
I don't follow how it's wrong, it's replicating exactly what `curUpdateTime` and `pervUpdateTime` used to do here, unless I'm missing something?

Reply:
see what i said in parenthesis. the time accounted for between `.ElapsedMilliseconds` and `.Restart()` is lost.

Reply:
Ah right, that is indeed wrong...

Reply:
Should be correct now I think.

Reply:
Why do you need the `Restart` call in the first place? as long as you're calling it there's still potential for error.

Reply:
Apparently another `Stopwatch` already exists for what seems to be clock throttling purposes, I've consumed its `tick` instead of what I've done.

Reply:
That linked code looks pretty bad for performance, but let's let that slide for now since it's out of scope. May be part of the reason android users report bad frame rates.